debian/changelog

```diff
@@ -1,3 +1,71 @@
+yaru-theme (19.10.1) eoan; urgency=medium
```

Comment:
Replace `eoan` with UNRELEASED, we can then do the changelog finalization + tagging once the package went in eoan for real.